OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -621,21 +621,25 @@ public static string GetExternalIP(Session.Client client, OrderManager orderMana
 			return address;
 		}
 
-		public static void SetupChatLine(ContainerWidget template, Color c, DateTime time, string from, string text)
+		public static void SetupChatLine(ContainerWidget template, DateTime time, string name1, Color nameColor,
+			string text,
+			Color textColor)
```

Comment:
Why is this spread out over two lines? Imho it would be better to have `string text, Color textColor)` in one line.
Having it all in one line (not three like at the moment) isn't even flagged as style error for me, so I'm not sure why you did this in the first place.